unity/Editor/Components/AbsoluteValuePropertyEditor.cs

```diff
@@ -1,3 +1,4 @@
+#if UNITY_EDITOR
```

Comment:
Could you please move these to underneath the copyright header? (Here and in other files)

Reply:
Thank you for the review, I've ensured the header comes first in all files.